src/Access/AccessControlHandler.php

```diff
@@ -77,7 +77,8 @@ public static function create(ContainerInterface $container) {
   public function hasPasswordChangeAccess(AccountInterface $current_user): AccessResultInterface {
     /** @var \Drupal\user\UserInterface $user */
     $user = $this->userStorage->load($current_user->id());
-    return AccessResult::allowedIf($user->get('password_enhancements_password_change_required')->getValue()[0]['value']);
+    $is_password_change_required = $user->get('password_enhancements_password_change_required')->getValue() ? (bool) $user->get('password_enhancements_password_change_required')->getValue()[0]['value'] : FALSE;
+    return AccessResult::allowedIf($is_password_change_required);
```

Comment:
Shouldn't this be `AccessResult::forbiddenIf(!$is_password_change_required)`?

Reply:
Not sure yet, in this PR I only wanted to fix the logic that causes WSOD. We'll see later, probably using `allowedIf` is fine because it only controls access to the `PasswordChangeForm`.

Reply:
Ok